feature/experimental/bgp/otg_tests/base_bgp_session_parameters/metadata.textproto

```diff
@@ -0,0 +1,44 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
```

Comment:
Sync metadata.textproto back to the current ATE version.

Reply:
@octpetre : Still needs another sync.

Reply:
Done

Reply:
corrected